bin/fuel-core/src/main.rs

```diff
@@ -1,15 +1,8 @@
-#![deny(clippy::arithmetic_side_effects)]
-#![deny(clippy::cast_possible_truncation)]
-#![deny(unused_crate_dependencies)]
-#![deny(warnings)]
```

Comment:
I think you can keep it in both places=)

Reply:
The first two don't affect the file at all. `#![deny(unused_crate_dependencies)]` will error about many crates that only the library depends on directly, and not the `main.rs`.